common/src/service/ExceptionTrace.cs

```diff
@@ -214,13 +213,13 @@ internal void BreakOnException(Exception exception)
                     {
                         Debugger.Launch();
                     }
-#elif !PCL && !NETSTANDARD1_3
+#elif !PCL && !NETSTANDARD1_3 && !NETSTANDARD2_0
```

Comment:
NET45 is not defined in the older csproj.  Will keep it as-is for now.  Ifdef cleanup will happen when netstandard1.3 release is official so that we could remove support for packages for individual platform.

---
In reply to: [141714190](https://github.com/Azure/azure-iot-sdk-csharp/pull/174#discussion_r141714190) [](ancestors = 141714190)

Reply:
Tracked by #176.